rskj-core/src/main/java/co/rsk/core/bc/BlockRelayValidatorImpl.java

```diff
@@ -66,7 +71,12 @@ public boolean isValid(@Nonnull Block block) {
         }
 
         Block parent = getParent(block);
-        return parent != null && blockParentValidator.isValid(block.getHeader(), parent);
+        if (parent == null) {
+            logger.error("Missing parent for block {}, cannot relay it", block.getNumber());
```

Comment:
`logger.warn()` ?

Reply:
In this case I decided to use error level because this shouldn't be happening in the moment this validator is run, but yeah, I guess that could change.